mlir/tools/mlir-miopen-driver/mlir-miopen-lib.hpp

```diff
@@ -1,15 +1,24 @@
+#include <cstddef>
+
 namespace mlir {
 
 typedef void *MlirHandle;
 
 extern "C" MlirHandle CreateMlirHandle(const char *options);
```

Comment:
Sounds good, between us we know that the lowering updates the mlir handle but it might not be intuitive enough for external developers. I will add some documentation explaining what each API does(return) and what is required for the arguments.